cpp/CMakeLists.txt

```diff
@@ -341,8 +341,9 @@ set(CUML_INCLUDE_DIRECTORIES
   ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
   ${SPDLOG_DIR}/src/spdlog/include
   ${FAISS_INCLUDE_DIRS}
+  ${RMM_INCLUDE_DIRS}
```

Comment:
This change is no longer needed (this lines don’t even exist as they were in this CMakeLists) now that we’re using the `RMM::RMM` target